### ISSUE TITLE
Reproducing stt_en_conformer_ctc_medium_ls training details

### ISSUE THREAD

**Issue by yiyibooks** [NVIDIA/NeMo#2525]
Hello,

I'm trying to train conformer_ctc_medium model from scratch. I use the config file from the pretrained model stt_en_conformer_ctc_medium_ls and only change the manifest_filepath and batch_size configuraion (I changed the batch_size to 16 to fit my GPU).

After training 42 epoches, the WER on librispeech validate data (about 15%) is still much higher than the pretrained stt_en_conformer_ctc_medium_ls model. 

May I know the trained epoch of the pretrained model stt_en_conformer_ctc_medium_ls ? Or is there any other configuration or training process I should pay attention to acheive similar results as the pretrained model ?

Thank you very much!

**Comment by VahidooX** [comment]
Your numbers looks OK to me. We used effective batch size of 2K which makes training faster but convergence slower. It makes it not easy to compare with yours. We trained that model for 1000 epochs on LS+Speech Perturbation. Are you training on just LS? 

Conformer models needs considerable number of epochs if you want to achieve SOTA numbers reported in papers. AFIK The original paper has trained the Conformer-Transducer for 1000 epochs. Conformer-CTC may need even more epochs as it uses CTC instead of Transducer decoding and it is non-autoregressive. If you need reasonable good accuracy, I think you may need to train at least 200 epochs.

**Comment by yiyibooks** [comment]
Thank you so much, dear VahidooX. 

I have trained on both LS and LS+spec_augment settings. Currently none of them are comparable with the pretrained model. I will continue the training as you suggest.
